tests/models/yolo/unit/test_yolo_config.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+import pytest
+
+from pl_bolts.models.detection.yolo.yolo_config import (
+    YOLOConfiguration,
+    _create_convolutional,
+    _create_maxpool,
+    _create_shortcut,
+    _create_upsample,
+)
+from tests import TEST_ROOT
+
+
+@pytest.mark.parametrize(
+    "config",
+    [
+        ({"batch_normalize": 1, "filters": 8, "size": 3, "stride": 1, "pad": 1, "activation": "leaky"}),
+        ({"batch_normalize": 0, "filters": 2, "size": 1, "stride": 1, "pad": 1, "activation": "mish"}),
+        ({"batch_normalize": 1, "filters": 6, "size": 3, "stride": 2, "pad": 1, "activation": "logistic"}),
+        ({"batch_normalize": 0, "filters": 4, "size": 3, "stride": 2, "pad": 0, "activation": "linear"}),
+    ],
+)
+def test_create_convolutional(config):
+    conv = _create_convolutional(config, [3])[0]
```

Comment:
I believe changing to `conv, _ = _create_convolutional(config, [3])` would increase readability a bit.
Same for all the other `_create_*[0]` methods.